src/api/public/apidocs-new/paths/person.yaml

```diff
@@ -38,3 +38,35 @@ get:
       $ref: '../components/responses/unauthorized.yaml'
   tags:
     - Person
+
+post:
+  summary: Executes a command on the person endpoint.
+  description: >-
+    Allows executing command on the person endpoint.
+    As of now, the only command allowed is 'register'.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: cmd
+      schema:
+        type: string
+      required: true
+      description: The command to execute
```

Comment:
It should be clear here, in the definition of the parameter, the possible options this parameter could have. Adding something like "The only command allowed is 'register'.", could be enough. Using an enum type, with only one value, would also help.

Reply:
I've used the union type

Reply:
I mean `enum`, not `union`